apps/src/lib/tools/jsinterpreter/CustomMarshaler.js

```diff
@@ -0,0 +1,69 @@
+
+module.exports = class CustomMarshaler {
```

Comment:
good question, will add a comment explaining. tl;dr: to make it work natively with node.js